build.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+#
+# shell script to build static elinks
+# for differenct architectures with
```

Comment:
I guess there is a typo in "differenct".

Reply:
I'm definitely ok with that. Above that I'd propose add elinks --version to info function.

Reply:
I;m not gonna reset my head. I tried to find way how to ammend commit and it always resulted in pull request automatic close.
I'll prepare fix together with some additional changes. I'd like to prepare test.sh for faster tests.